etc/init/rush-standalone.conf

```diff
@@ -0,0 +1,10 @@
+# auto start rush service if rush is installed standalone
+
+description "rush-standalone"
+
+start on filesystem or runlevel [2345]
+stop on shutdown
+
+script
+  [ ! -f /etc/init/spinnaker.conf ] && { start rush; }
+end script
```

Comment:
It is an interesting thought to use spinnaker.conf to auto start standalone services.
But it won't work without changing the way how spinnaker package works.
spinnaker package is not meant for standalone install. When install spinnaker package, all services are installed per dependency.
In a standalone deployment, each service is installed on a separate VM. For example, when front50 is installed standalone, no other services should be installed. In this case, spinnaker package is out of the picture.
thanks,
peter